packages/itinerary-body/src/stories/OtpUiItineraryBody.story.js

```diff
@@ -125,6 +129,10 @@ export const BikeTransitBikeItinerary = () => (
 export const WalkInterlinedTransitItinerary = () => (
   <ItineraryBodyDefaultsWrapper itinerary={walkInterlinedTransitItinerary} />
 );
+WalkInterlinedTransitItinerary.parameters = {
+  // Custom styling for this story only, not in production
+  a11y: { config: { rules: [{ id: "color-contrast", reviewOnFail: true }] } }
+};
```

Comment:
Extract the object assigned to `.parameters` to a `const`, and reuse.

Reply:
Done in 59cfcfb631e5a0f6c6ef52bc7260faf988bcd5bd